update_collections.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+COLLECTIONS = {"all" => 2062901859,
```

Comment:
I've seen this hardcoded hash twice, and that's enough to make me want to break it out.